srcpkgs/clamav/template

```diff
@@ -19,11 +19,11 @@ maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-only"
 homepage="https://www.clamav.net/"
 distfiles="https://www.clamav.net/downloads/production/clamav-${version}.tar.gz"
-checksum=def0ad15500fa6aff81d8e68b9f83aa75ee5b607a01335c1d26dbcc959932f85
+checksum=6f49da6ee927936de13d359e559d3944248e3a257d40b80b6c99ebe6fe8c8c3f
 _clamav_homedir="/var/lib/_${pkgname}"
 _clamav_descr="ClamAV user"
 system_accounts="_clamav"
-make_check=ci-skip
+make_check=ci-skip # I've no idea why...
```

Comment:
This comment should be dropped, it does not add any value to the template.

Reply:
Yeah, you are right, but the linter is complaining without it. I just wanted to make sure the critical update would be merged.

I'll drop the comment. :+1:

Reply:
I've fixed it. :+1: